### PR TITLE
AKS Migration — deploy review apps

### PR DESCRIPTION
**Pull request by malcolmbaig**
### Context

We're going to migrate AYTQ/CTR to AKS.

### Changes proposed in this pull request
This PR starts pulling together the files and configuration needed to manually deploy review apps to AKS.

It's a work-in-progress, opened as a draft PR for visibility and to hand over this work to @felixclack .

The resource group has been set up by @saliceti . The remaining work here can be driven out from:

- the error feedback received when running `make aks-review aks-terraform-plan PR_NUMBER=xx`
- using the files generated by the [new_service command](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/onboard-service.md#generate-code) in the Teacher Services Cloud repo as a reference, copying across required files as needed
- using Find a lost TRN as a reference for a Rails app with similar requirements running on AKS
- the specific technical requirements of the AYTQ/CTR services

### Link to Trello card
https://trello.com/c/Ng21PrA6/130-start-aytq-aks-migration-by-completing-onboarding-form-and-generating-initial-code

https://trello.com/c/OUi4acsm/132-%F0%9F%94%A5epic-design-and-implement-ctr-bulk-search-feature-for-hei-tsa (Epic)

### Checklist

- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Review by richardpattinson — APPROVED** [review]

File: terraform/application/terraform.tf [hunk 1]
```diff
@@ -0,0 +1,46 @@
+terraform {
+  required_version = "= 1.6.4"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "3.104.2"
+    }
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "2.24.0"
```
Comment (RMcVelia) [inline terraform/application/terraform.tf]: 2.32.0 is the latest, we have used 2.30.0 in recent services

File: terraform/application/variables.tf [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+variable "account_replication_type" {
+  description = "Replication LRS (across AZs) or GRS (across regions)"
+  default = "LRS"
+}
+variable "evidence_container_retention_in_days" {
+  description = "How long to keep the files"
+  default = 7
+  type = number
+}
```
Comment (RMcVelia) [inline terraform/application/variables.tf:68]: currently set for review apps?

Comment (saliceti) [reply terraform/application/variables.tf:68]: Updated the description and changed to 1 for review apps. Lifecycle for files should be implemented later

**Review by RMcVelia — APPROVED** [review]

**Comment by saliceti** [comment]
[AYTQ: Deploy review apps manually](https://trello.com/c/nvx9oNPs/1900-aytq-deploy-review-apps-manually)

**Pull request merged**